SuperChamps/contracts/NFTs/SCTempLockedNFT.sol

```diff
@@ -0,0 +1,124 @@
+// SPDX-License-Identifier: None
+// Super Champs Foundation 2024
+
+pragma solidity ^0.8.24;
+
+import "@openzeppelin/contracts/token/ERC721/ERC721.sol";
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import "./SCDefaultRenderer.sol";
+import "../../interfaces/IPermissionsManager.sol";
+import "../../interfaces/ISuperChampsToken.sol";
+import "../../interfaces/IERC721MetadataRenderer.sol";
+
+/// @title Super Champs (CHAMP) Temporarily Transfer Locked NFT
+/// @author Chance Santana-Wees (Coelacanth/Coel.eth)
+/// @dev Token transfers are restricted to addresses that have the Transer Admin permission until the CHAMP token is unlocked.
+/// @notice This is a standard ERC721 token contract that restricts token transfers before the protocol token is unlocked. Trades still possible via Shop/Marketplace system, but limited to sales denominated in CHAMP.
+contract SuperChampsToken is ERC721 {
```

Comment:
name confusing with SuperChampsToken rename to a bettername maybe

Reply:
This was a copy-pasta. You are correct it should be different.